Cargo.toml

```diff
@@ -38,14 +38,14 @@ members = [
 ]
 
 [dependencies]
-pravega-client-shared = { path = "./shared"}
-pravega-controller-client = { path = "./controller-client"}
-pravega-wire-protocol = { path = "./wire_protocol"}
-pravega-client-retry = {path = "./retry"}
-pravega-connection-pool = {path = "connection_pool" }
-pravega-client-channel = {path = "./channel"}
-pravega-client-auth = {path = "./auth"}
-pravega-client-config = {path = "./config"}
+pravega-client-shared = "0.1"
```

Comment:
Is there an option to put 0.1 in a variable? ; this would simplify work for future releases...

Reply:
That's a good suggestion but I don't think Cargo.toml file allows variable. It appears that they ruled out such suggestions.